examples/provisioner/windows.yaml

```diff
@@ -0,0 +1,57 @@
+# This example provisioner will provision instances running Windows Server OS
+
+apiVersion: karpenter.sh/v1alpha5
```

Comment:
The 2019 example and the 2022 example should probably be in different example files

Reply:
Got it, I will separate them.